src/jit/ee_il_dll.cpp

```diff
@@ -73,13 +73,19 @@ void __stdcall jitStartup(ICorJitHost* jitHost)
     if (jitstdout == nullptr)
     {
         int jitstdoutFd = _dup(_fileno(procstdout()));
```

Comment:
Should the check be applied to _fileno as well since it can invoke the invalid_param_handler as well (https://msdn.microsoft.com/en-us/library/zs6wbdhx.aspx)?

Also, I presume you tried the repro in https://github.com/dotnet/coreclr/issues/6356 and JIT started as expected?